setup.py

```diff
@@ -118,3 +118,8 @@
       entry_points=entry_points,
       **package_info
 )
+
+#Compiling the resource file for the gui
+import subprocess
```

Comment:
@wkerzendorf 
I have done what was done in the link you provided. And it seems to be working for me:

----------------------------------------------------------------------------------------------------------------------
 python setup.py develop
Freezing version number to tardis/version.py
running develop
running egg_info
creating tardis_sn.egg-info
writing requirements to tardis_sn.egg-info/requires.txt
writing tardis_sn.egg-info/PKG-INFO
writing top-level names to tardis_sn.egg-info/top_level.txt
writing dependency_links to tardis_sn.egg-info/dependency_links.txt
writing entry points to tardis_sn.egg-info/entry_points.txt
writing manifest file 'tardis_sn.egg-info/SOURCES.txt'
reading manifest file 'tardis_sn.egg-info/SOURCES.txt'
reading manifest template 'MANIFEST.in'
warning: no files found matching 'CHANGES.rst'
warning: no files found matching '*.c' under directory '*.pyx'
warning: no files found matching '*.pxd' under directory '*.pyx'
warning: no files found matching '*.h' under directory '*.pyx'
warning: no files found matching '*' under directory 'licenses'
warning: no files found matching '*' under directory 'cextern'
no previously-included directories found matching 'build'
no previously-included directories found matching 'docs/_build'
no previously-included directories found matching 'docs/api'
warning: no previously-included files matching '*.o' found anywhere in distribution
writing manifest file 'tardis_sn.egg-info/SOURCES.txt'
running build_ext
cythoning tardis/montecarlo/montecarlo.pyx to tardis/montecarlo/montecarlo.c
building 'tardis.montecarlo.montecarlo' extension
creating build
creating build/temp.linux-x86_64-2.7
creating build/temp.linux-x86_64-2.7/tardis
creating build/temp.linux-x86_64-2.7/tardis/montecarlo
creating build/temp.linux-x86_64-2.7/tardis/montecarlo/src
creating build/temp.linux-x86_64-2.7/tardis/montecarlo/src/randomkit
gcc -pthread -fno-strict-aliasing -g -O2 -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -fPIC -Itardis/montecarlo/src -Itardis/montecarlo/src/randomkit -I/home/gaurav/anaconda/envs/mytardisclonenv/lib/python2.7/site-packages/numpy/core/include -I/home/gaurav/anaconda/envs/mytardisclonenv/include/python2.7 -c tardis/montecarlo/montecarlo.c -o build/temp.linux-x86_64-2.7/tardis/montecarlo/montecarlo.o
In file included from /home/gaurav/anaconda/envs/mytardisclonenv/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:1804:0,
                 from /home/gaurav/anaconda/envs/mytardisclonenv/lib/python2.7/site-packages/numpy/core/include/numpy/ndarrayobject.h:17,
                 from /home/gaurav/anaconda/envs/mytardisclonenv/lib/python2.7/site-packages/numpy/core/include/numpy/arrayobject.h:4,
                 from tardis/montecarlo/montecarlo.c:250:
/home/gaurav/anaconda/envs/mytardisclonenv/lib/python2.7/site-packages/numpy/core/include/numpy/npy_1_7_deprecated_api.h:15:2: warning: #warning "Using deprecated NumPy API, disable it by " "#defining NPY_NO_DEPRECATED_API NPY_1_7_API_VERSION" [-Wcpp]
/home/gaurav/anaconda/envs/mytardisclonenv/lib/python2.7/site-packages/numpy/core/include/numpy/__ufunc_api.h:241:1: warning: ‘_import_umath’ defined but not used [-Wunused-function]
gcc -pthread -fno-strict-aliasing -g -O2 -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -fPIC -Itardis/montecarlo/src -Itardis/montecarlo/src/randomkit -I/home/gaurav/anaconda/envs/mytardisclonenv/lib/python2.7/site-packages/numpy/core/include -I/home/gaurav/anaconda/envs/mytardisclonenv/include/python2.7 -c tardis/montecarlo/src/cmontecarlo.c -o build/temp.linux-x86_64-2.7/tardis/montecarlo/src/cmontecarlo.o
tardis/montecarlo/src/cmontecarlo.c: In function ‘compute_distance2boundary’:
tardis/montecarlo/src/cmontecarlo.c:139:10: warning: unused variable ‘result’ [-Wunused-variable]
tardis/montecarlo/src/cmontecarlo.c: In function ‘compute_distance2line’:
tardis/montecarlo/src/cmontecarlo.c:225:4: warning: format ‘%d’ expects argument of type ‘int’, but argument 3 has type ‘int64_t’ [-Wformat]
tardis/montecarlo/src/cmontecarlo.c: In function ‘rpacket_init’:
tardis/montecarlo/src/cmontecarlo.c:681:8: warning: unused variable ‘close_line’ [-Wunused-variable]
tardis/montecarlo/src/cmontecarlo.c:672:10: warning: unused variable ‘nu_line’ [-Wunused-variable]
tardis/montecarlo/src/cmontecarlo.c: In function ‘binary_search’:
tardis/montecarlo/src/cmontecarlo.c:109:16: warning: ‘imid’ may be used uninitialized in this function [-Wuninitialized]
tardis/montecarlo/src/cmontecarlo.c: In function ‘montecarlo_compute_distances’:
tardis/montecarlo/src/cmontecarlo.c:915:18: warning: ‘d_line’ may be used uninitialized in this function [-Wuninitialized]
tardis/montecarlo/src/cmontecarlo.c:588:14: note: ‘d_line’ was declared here
tardis/montecarlo/src/cmontecarlo.c: In function ‘get_event_handler’:
tardis/montecarlo/src/cmontecarlo.c:915:18: warning: ‘d_line’ may be used uninitialized in this function [-Wuninitialized]
tardis/montecarlo/src/cmontecarlo.c:588:14: note: ‘d_line’ was declared here
tardis/montecarlo/src/cmontecarlo.c: In function ‘montecarlo_one_packet_loop’:
tardis/montecarlo/src/cmontecarlo.c:915:18: warning: ‘d_line’ may be used uninitialized in this function [-Wuninitialized]
tardis/montecarlo/src/cmontecarlo.c:588:14: note: ‘d_line’ was declared here
tardis/montecarlo/src/cmontecarlo.c: In function ‘montecarlo_one_packet’:
tardis/montecarlo/src/cmontecarlo.c:392:3: warning: ‘reabsorbed’ may be used uninitialized in this function [-Wuninitialized]
tardis/montecarlo/src/cmontecarlo.c:388:22: warning: ‘weight’ may be used uninitialized in this function [-Wuninitialized]
gcc -pthread -fno-strict-aliasing -g -O2 -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -fPIC -Itardis/montecarlo/src -Itardis/montecarlo/src/randomkit -I/home/gaurav/anaconda/envs/mytardisclonenv/lib/python2.7/site-packages/numpy/core/include -I/home/gaurav/anaconda/envs/mytardisclonenv/include/python2.7 -c tardis/montecarlo/src/randomkit/rk_isaac.c -o build/temp.linux-x86_64-2.7/tardis/montecarlo/src/randomkit/rk_isaac.o
gcc -pthread -fno-strict-aliasing -g -O2 -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -fPIC -Itardis/montecarlo/src -Itardis/montecarlo/src/randomkit -I/home/gaurav/anaconda/envs/mytardisclonenv/lib/python2.7/site-packages/numpy/core/include -I/home/gaurav/anaconda/envs/mytardisclonenv/include/python2.7 -c tardis/montecarlo/src/randomkit/rk_mt.c -o build/temp.linux-x86_64-2.7/tardis/montecarlo/src/randomkit/rk_mt.o
gcc -pthread -fno-strict-aliasing -g -O2 -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -fPIC -Itardis/montecarlo/src -Itardis/montecarlo/src/randomkit -I/home/gaurav/anaconda/envs/mytardisclonenv/lib/python2.7/site-packages/numpy/core/include -I/home/gaurav/anaconda/envs/mytardisclonenv/include/python2.7 -c tardis/montecarlo/src/randomkit/rk_sobol.c -o build/temp.linux-x86_64-2.7/tardis/montecarlo/src/randomkit/rk_sobol.o
gcc -pthread -fno-strict-aliasing -g -O2 -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -fPIC -Itardis/montecarlo/src -Itardis/montecarlo/src/randomkit -I/home/gaurav/anaconda/envs/mytardisclonenv/lib/python2.7/site-packages/numpy/core/include -I/home/gaurav/anaconda/envs/mytardisclonenv/include/python2.7 -c tardis/montecarlo/src/randomkit/rk_primitive.c -o build/temp.linux-x86_64-2.7/tardis/montecarlo/src/randomkit/rk_primitive.o
gcc -pthread -shared build/temp.linux-x86_64-2.7/tardis/montecarlo/montecarlo.o build/temp.linux-x86_64-2.7/tardis/montecarlo/src/cmontecarlo.o build/temp.linux-x86_64-2.7/tardis/montecarlo/src/randomkit/rk_isaac.o build/temp.linux-x86_64-2.7/tardis/montecarlo/src/randomkit/rk_mt.o build/temp.linux-x86_64-2.7/tardis/montecarlo/src/randomkit/rk_sobol.o build/temp.linux-x86_64-2.7/tardis/montecarlo/src/randomkit/rk_primitive.o -L/home/gaurav/anaconda/envs/mytardisclonenv/lib -lpython2.7 -o /home/gaurav/GSoC/installationtesting/tardis/tardis/montecarlo/montecarlo.so
cythoning tardis/macro_atom.pyx to tardis/macro_atom.c
building 'tardis.macro_atom' extension
gcc -pthread -fno-strict-aliasing -g -O2 -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -fPIC -I/home/gaurav/anaconda/envs/mytardisclonenv/lib/python2.7/site-packages/numpy/core/include -I/home/gaurav/anaconda/envs/mytardisclonenv/include/python2.7 -c tardis/macro_atom.c -o build/temp.linux-x86_64-2.7/tardis/macro_atom.o
In file included from /home/gaurav/anaconda/envs/mytardisclonenv/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:1804:0,
                 from /home/gaurav/anaconda/envs/mytardisclonenv/lib/python2.7/site-packages/numpy/core/include/numpy/ndarrayobject.h:17,
                 from /home/gaurav/anaconda/envs/mytardisclonenv/lib/python2.7/site-packages/numpy/core/include/numpy/arrayobject.h:4,
                 from tardis/macro_atom.c:250:
/home/gaurav/anaconda/envs/mytardisclonenv/lib/python2.7/site-packages/numpy/core/include/numpy/npy_1_7_deprecated_api.h:15:2: warning: #warning "Using deprecated NumPy API, disable it by " "#defining NPY_NO_DEPRECATED_API NPY_1_7_API_VERSION" [-Wcpp]
/home/gaurav/anaconda/envs/mytardisclonenv/lib/python2.7/site-packages/numpy/core/include/numpy/__multiarray_api.h:1629:1: warning: ‘_import_array’ defined but not used [-Wunused-function]
/home/gaurav/anaconda/envs/mytardisclonenv/lib/python2.7/site-packages/numpy/core/include/numpy/__ufunc_api.h:241:1: warning: ‘_import_umath’ defined but not used [-Wunused-function]
gcc -pthread -shared build/temp.linux-x86_64-2.7/tardis/macro_atom.o -L/home/gaurav/anaconda/envs/mytardisclonenv/lib -lpython2.7 -o /home/gaurav/GSoC/installationtesting/tardis/tardis/macro_atom.so
Creating /home/gaurav/anaconda/envs/mytardisclonenv/lib/python2.7/site-packages/tardis-sn.egg-link (link to .)
Removing tardis-sn 1.0.1 from easy-install.pth file
Adding tardis-sn 1.0.1 to easy-install.pth file
Installing tardis script to /home/gaurav/anaconda/envs/mytardisclonenv/bin

Installed /home/gaurav/GSoC/installationtesting/tardis
Processing dependencies for tardis-sn==1.0.1
Searching for astropy==1.0
Best match: astropy 1.0
Adding astropy 1.0 to easy-install.pth file

Using /home/gaurav/anaconda/envs/mytardisclonenv/lib/python2.7/site-packages
Finished processing dependencies for tardis-sn==1.0.1
-----------------------------------------------------------------------------------------------------------------------

Your error seems to be saying that a file or directory is missing. Did you clone from 
git@github.com:gautam1168/tardis.git ?